include/deal.II/hp/dof_handler.h

```diff
@@ -182,7 +182,7 @@ namespace hp
      *
      * @ingroup Iterators
      */
-    typedef typename ActiveSelector::cell_iterator        cell_iterator;
+    typedef TriaIterator < dealii::DoFCellAccessor < DoFHandler < dim, spacedim >, false > >        cell_iterator;
```

Comment:
These changes I can live with, though it may be worthwhile just replacing all of the uses of `ActiveSelector` and `LevelSelector` in this case.